common/commands/command.go

```diff
@@ -43,32 +50,45 @@ func reportUsage(command Command, channel chan<- bool) {
 
 	serverDetails, err := command.ServerDetails()
 	if err != nil {
-		log.Debug("Usage reporting:", err.Error())
+		log.Debug("Usage reporting. Failed accessing ServerDetails.", err.Error())
+		return
+	}
+	if serverDetails == nil || serverDetails.ArtifactoryUrl == "" {
+		return
+	}
+	serviceManager, err := utils.CreateServiceManager(serverDetails, -1, 0, false)
+	if err != nil {
+		log.Debug("Usage reporting. Failed creating the Artifactory Service Manager.", err.Error())
+		return
+	}
+	ArtifactoryVersion, err := serviceManager.GetVersion()
+	if err != nil {
+		log.Debug("Usage reporting. Failed getting the Artifactory", err.Error())
```

Comment:
```suggestion
		log.Debug("Usage reporting: ", err.Error())
```
GetVersion errors are already wrapped and explaining the issue